daemon/cmd/daemon_main.go

```diff
@@ -216,9 +216,12 @@ func init() {
 	})
 
 	// Env bindings
-	flags.Int(option.AgentHealthPort, defaults.AgentHealthPort, "TCP port for agent health status API")
+	flags.Int(option.AgentHealthPort, defaults.AgentHealthPort, "TCP port for agent health status API inside agent pod")
 	option.BindEnv(option.AgentHealthPort)
 
+	flags.Int(option.ClusterHealthPort, defaults.ClusterHealthPort, "TCP port for cluster-wide health API")
```

Comment:
There is a subtlety here with the other new option: `cilium-health` is an entirely different kind of health API vs. what is commonly found in k8s around the healthiness of individual applications. It is a network connectivity check really. To try to minimize confusion, I'd be inclined to try to represent that somehow in the flag description here. Maybe something like:

```
flags.Int(option.ClusterHealthPort, defaults.ClusterHealthPort, "TCP port for cluster-wide network connectivity health API")
```

Reply:
Thanks, Joe! I'll go with that :)